docs/root/version_history/current.rst

```diff
@@ -7,6 +7,7 @@ Incompatible Behavior Changes
 *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
 
 * build: official released binary is now built on Ubuntu 18.04, requires glibc >= 2.27.
+* router: path_redirect now keeps query string by default.
```

Comment:
I think even if we think this is the right thing to do, this needs to be runtime guarded (default can be on). Can you please add a runtime feature flag for this?

Reply:
I am not sure how to add a runtime feature flag and what its name should be, could you please provide an example?

Reply:
Take a look at some blame/commits that touch this array: https://github.com/envoyproxy/envoy/blob/master/source/common/runtime/runtime_features.cc#L54